salt/states/user.py

```diff
@@ -262,6 +263,10 @@ def present(name,
     .. versionchanged:: 0.16.0
        BSD support added.
 
+    hash_password
+        Set to True to encrypt the clear text password. Default is ``False``.
```

Comment:
This language seems misleading to me. What we're doing here is *hashing* a password -- not encrypting it. Could we change it so that we're totally clear about what's being done?